src/lib/components/wizard-navigation-bar.component.spec.ts

```diff
@@ -475,10 +469,7 @@ describe('WizardNavigationBarComponent', () => {
     wizardTest.wizard.navBarLayout = 'large-filled';
     wizardTestFixture.detectChanges();
 
-    expect(navBarEl.classes).toEqual({
-      'horizontal': true, 'vertical': false, 'small': false,
-      'large-filled': true, 'large-filled-symbols': false, 'large-empty': false, 'large-empty-symbols': false
-    });
+    expect(navBarEl.classes).toEqual({ 'horizontal': true, 'large-filled': true });
```

Comment:
It seems like the contract of `classes` has changed. The returned object now only contains the active classes, i.e. all classes that are true. This change affected 11 tests, which were "fixed" by me.